package.json

```diff
@@ -77,7 +77,7 @@
     "ts-node": "^3.2.0",
     "tslint": "^4.5.1",
     "tslint-microsoft-contrib": "^4.0.1",
-    "typescript": "2.4.1",
+    "typescript": "^2.4.2",
```

Comment:
I _think_ we generally wanna pin to an exact version of TS, or at least make this `~` instead of `^`, since TS can change pretty significantly in "minor" releases.